pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.black]
 line-length = 120
-target-version = ['py37', 'py38']
+target-version = ['py37', 'py38','py39']
```

```suggestion
target-version = ['py37', 'py38', 'py39']
```
please keep the same formatting with spaces.

---

This is unfortunately not possible with fixed version of black we are using